### PR TITLE
Feature/sc 5674 refactoring homework

### PR DESCRIPTION
**Pull request by TomaszJuchimiuk**
# Description
This new PR after correction https://github.com/hpi-schul-cloud/integration-tests/pull/111
Also moved all methods from creatTask.js to new POs.
Adapted tests to new POs for Courses.

## Approval for review
- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: page-objects/pages/HMWRKAddEditHomeworkPage.js [hunk 1]
```diff
@@ -1 +1,95 @@
-/*[url/homework/new] | [url/homework/[homeworkId]/edit]*/
\ No newline at end of file
+/*[url/homework/new] | [url/homework/[homeworkId]/edit]*/
+'use strict';
+const waitHelpers = require('../../runtime/helpers/waitHelpers.js');
+const dateTimeHelpers = require('../../runtime/helpers/dateTimeHelpers.js');
+const courseData = require('../../shared-objects/courseData');
+const courseHomeworksPage = require("../pages/coursePages/CRSSCourseHomeworksPage.js");
+
+const taskNameField = "input[placeholder='Titel']";
```
Comment (yuliashikhareva) [inline page-objects/pages/HMWRKAddEditHomeworkPage.js:8]: let us unite selectors in const selectors = {selector1: 'xxx',}

File: page-objects/pages/HMWRKAddEditHomeworkPage.js [hunk 2]
```diff
@@ -1 +1,95 @@
...
+const submitTaskButton = ".btn-submit";
```
Comment (yuliashikhareva) [inline page-objects/pages/HMWRKAddEditHomeworkPage.js:9]: the same, please add to selectors

File: src.code-workspace [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+	"folders": [
```
Comment (yuliashikhareva) [inline src.code-workspace:2]: what is this?

File: page-objects/pages/HMWRKAddEditHomeworkPage.js (hunk 1 above)
Comment (TomaszJuchimiuk) [reply page-objects/pages/HMWRKAddEditHomeworkPage.js]: done

File: page-objects/pages/HMWRKAddEditHomeworkPage.js (hunk 2 above)
Comment (TomaszJuchimiuk) [reply page-objects/pages/HMWRKAddEditHomeworkPage.js]: done

File: src.code-workspace (hunk 3 above)
Comment (TomaszJuchimiuk) [reply src.code-workspace]: deleted.

**Pull request merged**